test/StringUtilsTest.cpp

```diff
@@ -211,6 +211,45 @@ TEST(StringUtilsTest, strip) {
   ASSERT_EQ(u8"äö", strip(u8"xxaxaxaxaäöaaaxxx", "xa"));
   ASSERT_EQ("xxaxaxaxa♥", rstrip("xxaxaxaxa♥aaaxxx", "xa"));
 }
+
+
+TEST(StringUtilsTest, splitWs) {
+  string s1 = "  this\nis\t  \nit  ";
+  string s2 = "\n   \t  \n \t";
+  string s3 = "thisisit";
+  string s4 = "this is\nit";
+  string s5 = "a";
```

Comment:
Can you add a test with unicode chars here, just to be sure we don't break that, considering the signed/unsigned char mess ⇒ int mess

Reply:
This of course also needs `setlocale(LC_CTYPE, "en_US.utf8");`